SpringProject/src/test/java/com/gradproject2019/UserConferenceServiceTest.java

```diff
@@ -48,4 +50,18 @@ public void shouldSaveInterestInConference() {
         assertThat(userConference.getUserId()).isEqualTo(userConferenceResponseDto.getUserId());
 
     }
+
+    @Test
+    public void shouldGetInterestInConference() {
```

Comment:
I think you could probably add a few more test cases here, this seems like a good happy path test but there are a few unhappy paths that are untested